repair_type/readme/ROADMAP.rst

```diff
@@ -0,0 +1,7 @@
+Roadmap
+~~~~~~~
```

Comment:
The title of the fragments will be automatically included in the readme by the generator. If you add it manually it will be duplicated.